addons/l10n_it_stock_ddt/report/l10n_it_ddt_report.xml

```diff
@@ -124,7 +124,12 @@
                                         <span t-field="move.product_uom" groups="uom.group_uom"/>
                                     </td>
                                     <td>
-                                        <t t-set="lst_price" t-value="move.product_id.lst_price * move.quantity_done"/>
+                                        <t t-if="move.sale_line_id">
+                                            <t t-set="lst_price" t-value="move.sale_line_id.price_subtotal"/>
```

Comment:
This should be proportional to the amounts in the sale line.  (see https://github.com/odoo/enterprise/blob/c11cda2dc24dac731054c0e2c26adabb39ffb753/l10n_cl_edi_stock/models/stock_picking.py#L278 )